Api/Data/PlaceOrder/Request/OrderDataInterface.php

```diff
@@ -32,7 +32,6 @@ interface OrderDataInterface
         self::PROPERTY_ORDER_NUMBER,
         self::PROPERTY_TOTAL,
         self::PROPERTY_PAYMENT,
-        self::PROPERTY_TRANSACTION,
```

Comment:
Question: This is for my personal knowledge what are those property could you direct me to any documentation?